tests/foreman/cli/test_hammer.py

```diff
@@ -164,26 +154,26 @@ def test_positive_disable_hammer_defaults():
     try:
         Defaults.add({'param-name': 'organization_id', 'param-value': default_org['id']})
         # list templates for BZ#1368173
-        result = ssh.command('hammer job-template list')
-        assert result.return_code == 0
+        result = default_sat.execute('hammer job-template list')
+        assert result.status == 0
         # Verify --organization-id is not required to pass if defaults are set
-        result = ssh.command('hammer product list')
-        assert result.return_code == 0
+        result = default_sat.execute('hammer product list')
```

Comment:
all these all hammer calls can be converted as https://github.com/SatelliteQE/robottelo/pull/8928/files#diff-6a4124eaa47ca4e4b1dc1a9f6716582f1161970568444d0a2a698a410186b318R211 (non-blocking)

Reply:
they sure can, but I will leave further refactoring of these tests to @Gauravtalreja1